app/models/spree/license_key.rb

```diff
@@ -7,6 +7,15 @@ class LicenseKey < ActiveRecord::Base
     attr_accessible :license_key, :inventory_unit_id, :variant_id, :void, :memo, :activated_on
 
     scope :available, where(inventory_unit_id: nil, void: false)
-    scope :used, where('(inventory_unit_id IS NOT NULL) OR (void = ?)', true)
+    scope :used, where('inventory_unit_id IS NOT NULL')
+    scope :void, where(void: true)
+
+    def used?
+      self.class.used.where(id: id).any?
```

Comment:
That is a good point.  I think I was just disappointed that the scope didn't automatically provide this method, so I went out of my way to write it this way.  Do you think that the logic duplication is okay to have?